.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    labels:
+      - type:ci
```

```suggestion
      - type:ci
    # https://til.simonwillison.net/github/dependabot-python-setup
    groups:
      github-actions:
        patterns:
          - '*'
```
И сюда тоже добавим группировку, чтобы dependabot не генерил по отдельному MR на каждый отдельный action. В остальные репы я потом добавлю те же правки